src/pages/using-spark/guides/alt-text.mdx

```diff
@@ -0,0 +1,352 @@
+
+### Images
+
+Not all images on the web need the same alt-text. It’s important to consider the context and purpose of the image when writing the text description. 
+
+Images can be active, informative or decorative.
+
+- **Active Images** help communicate meaning, actions, status or feedback.
+- **Informative Images** help users understand the content on a screen. These include: diagrams, charts, graphs, maps or informative icons.
+- **Decorative Images** have no semantic meaning and should have their alt attributes set to the empty (or null) string ("").
+
+> For more information refer to <a href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt">Developer Mozilla's HTML Image Element - alt</a>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+#### Guidelines
+
+<div class="sprk-b-TableContainer" data-id="table-2">
+  <table class="sprk-b-Table--secondary sprk-b-Table--spacing-medium">
+    <thead>
+      <tr>
+        <th>If The Image Is ...</th>
+        <th>How To Identify It?</th>
+        <th>The Alt Text Should ...</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Active</td>
+        <td>These images perform an action or have functionality. For example, clickable icons are active images.</td>
+        <td>Help the user understand what happens when they click the image.</td>
+      </tr>
+      <tr>
+        <td>Informative</td>
+        <td>These images contain information the user may need. If the image was removed from the screen, the information would be lost.</td>
+        <td>Explain or describe the information found in the image.</td>
+      </tr>
+      <tr>
+        <td>Decorative</td>
+        <td>These images are included only for placement or visual effects. An image is decorative if you could remove it from the screen without losing information. </td>
+        <td>Blank or null ("")</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvm" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbl sprk-u-Measure">Icons</h3>
+
+<p class="sprk-b-TypeBodyTwo sprk-u-mbl sprk-u-Measure">Icons can potentially convey meaning without using words, although researching users understanding of icons is critical to ensuring that meaning is clear.</p>
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbs sprk-u-Measure">Guidelines</h3>
+
+<div class="sprk-b-TableContainer" data-id="table-2">
+  <table class="sprk-b-Table--secondary sprk-b-Table--spacing-medium">
+    <thead>
+      <tr>
+        <th>If The Icon Is ...</th>
+        <th>How To Identify It?</th>
+        <th>Alt Text Should ...</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Added as an inline image</td>
+        <td>An icon that appears as an inline image, <strong>must</strong> have an alt attribute.</td>
+        <td>Help the user understand what the icon image conveys</td>
+      </tr>
+      <tr>
+        <td>For decoration</td>
```

Comment:
Some of these have periods at the end and others don't. Should they all have one?